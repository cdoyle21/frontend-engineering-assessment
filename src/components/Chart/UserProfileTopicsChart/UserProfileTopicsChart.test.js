import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import React from 'react';
import UserProfileTopicsChart from './UserProfileTopicsChart';

configure({ adapter: new Adapter() });

describe('Render UserProfileTopicsChart component', () => {
    it('should be defined', () => {
        expect(UserProfileTopicsChart).toBeDefined();
      });
    it('should render UserProfileTopicsChart component correctly', () => {
        const wrapper = shallow(
        <UserProfileTopicsChart />,
        );
        expect(wrapper).toMatchSnapshot();
    });
  });
