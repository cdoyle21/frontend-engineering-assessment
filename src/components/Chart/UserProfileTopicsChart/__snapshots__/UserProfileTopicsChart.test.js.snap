// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render UserProfileTopicsChart component should render UserProfileTopicsChart component correctly 1`] = `ShallowWrapper {}`;
